layouts/_default/baseof.html

```diff
@@ -43,6 +43,9 @@
 		{{ partial "footer" . }}
 	</div>
 <script async defer src="{{ "js/menu.js" | relURL }}"></script>
+{{ range .Site.Params.customJS -}}
+<script async defer src="{{ . | relURL }}"></script>
```

Comment:
Hello, @oshliaer 

I do not think that `async defer` is suitable for all cases. Asynchronous loading for third-party scripts can break something.

Reply:
Can I add properties? The array of strings is not too informative.

Reply:
I've deleted this.

Reply:
> Can I add properties? The array of strings is not too informative.

Sorry, I didn’t quite catch that. What do you mean?